{{cookiecutter.project_shortname}}/README.md

```diff
@@ -0,0 +1,86 @@
+    _Note: venv\Scripts\activate for windows_
+
+3. Install python packages required to build components.
+    ```
+    $ pip install -r requirements.txt
+    ```
+4. Install the python packages for testing (optional)
+    ```
+    $ pip install -r tests/requirements.txt
+    ```
+
+### Write your component code in `src/lib/components/{{cookiecutter.component_name}}.react.js`. 
+
+- The demo app is in `src/demo` and you will import your example component code into your demo app.
+- Test your code in a Python environment:
+    1. Build your code
+        ```
+        $ npm run build:all
+        ```
+    2. Run and modify the `usage.py` sample dash app:
+        ```
+        $ python usage.py
+        ```
+- Write tests for your component.
+    - A sample test is available in `tests/test_usage.py`, it will load `usage.py` and you can then automate interactions with selenium.
+    - Run the tests with `$ pytest tests`.
+    - The Dash team uses these types of integration tests extensively. Browse the Dash component code on GitHub for more examples of testing (e.g. https://github.com/plotly/dash-core-components)
+- Add custom styles to your component by putting your custom CSS files into your distribution folder (`{{cookiecutter.project_shortname}}`).
+    - Make sure that they are referenced in `MANIFEST.in` so that they get properly included when you're ready to publish your component.
+    - Make sure the stylesheets are added to the `_css_dist` dict in `{{cookiecutter.project_shortname}}/__init__.py` so dash will serve them automatically when the component suite is requested.
+- [Review your code](./review_checklist.md)
+
+### Create a production build and publish:
+
+1. Build your code:
+    ```
+    $ npm run build:all
+    ```
+2. Create a Python tarball
+    ```
+    $ python setup.py sdist
+    ```
+    This distribution tarball will get generated in the `dist/` folder
+
+3. Test your tarball by copying it into a new environment and installing it locally:
+    ```
+    $ pip install my_dash_component-0.0.1.tar.gz
+    ```
+
+4. If it works, then you can publish the component to NPM and PyPI:
+    1. Cleanup the dist folder (optional)
+        ```
+        $ rm -rf dist
+        ```
+    2. Publish on PyPI
+        ```
+        $ twine upload dist/*
+        ```
+    3. Publish on NPM (Optional if chosen False in `publish_on_npm`)
+        ```
+        $ npm publish
+        ```
+        _Publishing your component to NPM will make the JavaScript bundles available on the unpkg CDN. By default, Dash servers the component library's CSS and JS from the remote unpkg CDN, so if you haven't published the component package to NPM you'll need to set the `serve_locally` flags to `True` (unless you choose `False` on `publish_on_npm`). We will eventually make `serve_locally=True` the default, [follow our progress in this issue](https://github.com/plotly/dash/issues/284)._
+5. Share your component with the community! https://community.plot.ly/c/dash
```

Comment:
I'd like to update this with:

5. Share your component with the community!
    1. Publish this repository to GitHub
    2. Tag your GitHub repository with the `plotly-dash` tag so that it appears here: https://github.com/topics/plotly-dash
    3. Create a post in the Dash community forum: https://community.plot.ly/c/dash